package/com.unity.formats.usd/Samples/ImportMaterials/Unity.Formats.USD.ImportMaterials.asmdef

```diff
@@ -1,6 +1,7 @@
 {
     "name": "Unity.Formats.USD.ImportMaterials",
     "references": [
+        "Unity.Formats.USD.Editor",
```

Comment:
I was wondering what Editor function are you using in this assembly?
To test if you are able to make a build, with samples.

Reply:
I use the path function I introduced.

Reply:
Build was failing because of a missing assembly 🤦 Now fixed.